### ISSUE TITLE
[ListView] the keyboard navigation doesn't work as expected with Angular 10

### ISSUE THREAD

**Issue by svetq** [telerik/kendo-angular#3128]
**Describe the bug**
When using a navigable ListView in Angular 10, then pressing the down arrow key doesn't focus the following item.

**To Reproduce**
Please check the attached example project:
[list_view.zip](https://github.com/telerik/kendo-angular/files/5603032/list_view.zip)

**Comment by mcr85** [comment]
Same happens for me as in @svetq's example. It seems navigating up works but down doesn't.
`keydown` event handler needs `listViewItems` QueryList length and it seems to be empty.
ListView examples on StackBlitz work though. I can't see difference between @svetq's and ListView StackBlitz examples.
Can you assist please?

**Comment by dimitar-pechev** [comment]
@mcr85  The problem is due to a misconfiguration of the `ContentChildren` query in our code which becomes evident only in newer angular version. The examples on our side run on older angular versions for backwards compatibility reasons and that's why the problem is not present there.

The issue is already identified and the fix is on a PR, so it should soon be available.

**Comment by tscislo** [comment]
We also have this problem in out app. When will you release new version @dimitar-pechev ?

**Comment by dimitar-pechev** [comment]
The fix is available in the latest [dev version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/) of the package - v1.0.0-dev.202012091736. An official version will be released next week as part of https://github.com/telerik/kendo-angular/issues/3144.

**Comment by dimitar-pechev** [comment]
Fixed in **v1.0.0** of the ListView package.
